src/test/java/uk/gov/hmcts/reform/hmc/service/CftHearingServiceImplTest.java

```diff
@@ -134,8 +140,40 @@ void shouldFailToValidateCaseWhenConnectivityIssue() {
         doThrow(exception).when(restTemplate).exchange(anyString(), eq(HttpMethod.GET), any(HttpEntity.class),
                                                        eq(HttpStatus.class));
         final ServiceException expectedException =
-            assertThrows(ServiceException.class, () -> cftHearingService.getLatestVersion(validCaseId));
+            assertThrows(ServiceException.class, () -> cftHearingService.getHearingVersionHeaders(validCaseId));
         assertEquals("The CFT service is currently down, please refresh "
                          +  "your browser or try again later", expectedException.getMessage());
     }
+
+    @Test
+    void shouldFailAsHearingInTerminalState_Completed() {
+        responseHeaders.set("Latest-Hearing-Status","COMPLETED");
```

Comment:
Could use constants instead of all these hard-coded fields, especially for status types